docs/howto/modifying_tags_and_unavailability.rst

```diff
@@ -0,0 +1,45 @@
+Modifying event tags and unavailabiity.
```

Comment:
Can we just change this so that all the titles complete the sentence 'How to ....'

So, in this case it would be 'Modify tags and unavailability'